src/anyio/_backends/_asyncio.py

```diff
@@ -100,8 +100,8 @@ def _cancel_all_tasks(loop):
                 events.set_event_loop(None)
                 loop.close()
 
-    def create_task(coro: Union[Generator[Any, None, _T], Awaitable[_T]], *,  # type: ignore
-                    name: Optional[str] = None) -> asyncio.Task:
+    def create_task(coro: Union[Generator[Any, None, _T], Awaitable[_T]], *,
+                    name: object = None) -> asyncio.Task:
```

Comment:
why `object` here ? just for my knowledge

Reply:
To match typeshed https://github.com/python/typeshed/pull/5494

Reply:
woo! ok thanks